omero/sysadmins/mail.txt

```diff
@@ -1,12 +1,17 @@
 OMERO.mail
 ==========
 
-The OMERO server has the ability to send email to any users who have a properly
-configured email address. OMERO system administrators can then use the
-:omerocmd:`admin email` command to contact those users.
+The OMERO server has the ability to send email to any users who have a
+properly configured email address. OMERO system administrators can then use
+the :omerocmd:`admin email` command to contact those users.
 
-In order to activate the subsystem, minimally the :property:`omero.mail.config`
-property will need to be activated.
+In order to activate the subsystem, minimally the
+:property:`omero.mail.config` property will need to be activated.
+
+.. note:: A current limitation of the system is that emails are not in a queue
+    and therefore if you log out or otherwise lose your OMERO.web session
```

Comment:
Does `omero admin email` work on the CLI too? If so, we might want to reword "if you log out or otherwise lose your OMERO.web session" to remove the `.web` bit.